src/IntroPage/index.js

```diff
@@ -67,7 +67,7 @@ const style = css`
     margin-left: 182px;
     line-height: 1.4;
   }
-  @media only screen and (min-width: 320px) and (max-width: 480px) {
+  @media only screen and (max-width: 480px) {
```

Comment:
since we don't have designs for landscape mode yet, what's better to show in landscape mode, mobile or desktop?

Reply:
Desktop if landscape screen has enough width. Mobile otherwise.